main.nf

```diff
@@ -105,10 +99,40 @@ Duration    : ${workflow.duration}
 Success     : ${workflow.success}
 workDir     : ${workflow.workDir}
 Exit status : ${workflow.exitStatus}
-outDir      : ${params.outDir}
+outDir      : ${params.output}
 
 =======================================================================================
   """
 println summary
+}
+// ====================================================================================
+
+process fastqc {
+    // Specify resouce allocation for this process
+    cpus 2
+    memory '12 GB'
+    container 'quay.io/biocontainers/fastqc:0.12.1--hdfd78af_0'
+    publishDir "${params.output}", mode: 'symlink'
```

Comment:
Publish outputs to your specified outdir with `--output`

Reply:
Can change mode to copy instead of symlink if needed. See [here](https://nextflow.io/docs/edge/process.html#publishdir)